tests/functional/z_cluster/test_performance_profile_validation.py

```diff
@@ -0,0 +1,159 @@
+import logging
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import (
+    tier4a,
+    skipif_ocs_version,
+    brown_squad,
+)
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    skipif_external_mode,
+    skipif_managed_service,
+)
+from ocs_ci.framework import config
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.storage_cluster import verify_storage_cluster
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import (
+    get_pods_having_label,
+    Pod,
+)
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.ocs.resources.storage_cluster import StorageCluster
+
+
+log = logging.getLogger(__name__)
+
+
+@brown_squad
+@tier4a
+@skipif_external_mode
+@skipif_managed_service
+@skipif_ocs_version("<4.15")
+@pytest.mark.polarion_id("XXXX")
```

Comment:
Planing to create TC in polrion or waiting for polarion data move?

Reply:
Yes @ramkiperiy

Reply:
Do add the polarion ID, its generated and updated in test plan.